docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
```

Comment:
Do we want to start off the home page by sending people elsewhere if they're interested in anything other than console apps?  Chances are they came here from there.  Consider starting off by introducing the framework/platform first?  This part seems to me to fit better under a Getting Started section.